pkg/controllers/dynakube/activegate/reconciler_test.go

```diff
@@ -478,7 +483,7 @@ func TestServiceCreation(t *testing.T) {
 		}
 	})
 
-	t.Run("service exposes correct ports for multiple capabilities", func(t *testing.T) {
+	t.Run("service exposes all ports for all capabilities", func(t *testing.T) {
```

Comment:
this test duplicates `t.Run("service exposes all ports for every capabilities", func(t *testing.T)` line 438.
It checks open ports for Routing capability - so can be deleted.